### PR TITLE
SAK-45166 Adjust font size of Assignments Honor Pledge

### PR DESCRIPTION
**Pull request by V100X**

### CODE REVIEW

**Comment by V100X** [comment]
SAK-45166

**Comment by mpellicer** [comment]
Hi @V100X , would you mind signing in this ICLA? 

https://www.apereo.org/licensing/agreements/icla

It's the individual contributor license agreement.

Thanks

File: library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_jumbotron.scss [hunk 1]
```diff
@@ -17,8 +17,8 @@
 
   p {
     margin-bottom: ($jumbotron-padding / 2);
-    font-size: $jumbotron-font-size;
-    font-weight: 200;
+    font-size: $default-font-size;
```
Comment (josecebe) [inline library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_jumbotron.scss:20]: Now "$jumbotron-font-size" variable is unused, maybe its new value should be $default-font-size?

File: library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss [hunk 2]
```diff
@@ -555,7 +555,7 @@
 
 	#honor-pledge-agreement {
 		h4 {
-			font-size: $jumbotron-heading-font-size;
```
Comment (josecebe) [inline library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss:558]: Maybe "$jumbotron-heading-font-size" new value should be $font-size-h4?

**Review by josecebe — CHANGES_REQUESTED** [review]

File: library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss (hunk 2 above)
Comment (josecebe) [reply library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss:558]: After the change, check other places where $jumbotron-heading-font-size is used

**Review by josecebe — APPROVED** [review]
LGTM

**Review by ern — APPROVED** [review]

**Pull request merged**